frontend/src/pages/about/hmi.js

```diff
@@ -177,18 +177,24 @@ const HmiPage = ({ pageContext }) => {
       description:
         "T & J were sent out from HMCC of Ann Arbor to serve in an East Asian nation as missionaries in 2024",
     },
-  ].reduce(
-    (rows, key, index) =>
-      (index % 4 === 0 ? rows.push([key]) : rows[rows.length - 1].push(key)) &&
-      rows,
-    []
-  );
-
-  const project_cols = 2;
-  const project_lg_cols = 3;
-  let project_rows = (projects.length + project_cols - 1) / project_cols;
-  let project_lg_rows =
-    (projects.length + project_lg_cols - 1) / project_lg_cols;
```

Comment:
why remove this?

Reply:
oh I see, I think I added this in but forgot to actually change the tailwind tags in the `projects.map` section. It's mean to dynamically edit the number of rows necessary in the flags grid so that if we add more countries/projects in the future, it can auto format.

Reply:
Here is what the final UI/UX for mission projects:
![image](https://github.com/user-attachments/assets/d8539157-3e88-45de-9c37-bc8b18628bf1)